main.go

```diff
@@ -123,17 +142,131 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 			nil,
 		),
 		evictions: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
+			prometheus.BuildFQName(namespace, "", "evicted_total"),
 			"Number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
-			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
+			prometheus.BuildFQName(namespace, "", "reclaimed_total"),
 			"Number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		active_slabs: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "active_slabs_total"),
```

Comment:
This is a gauge and should not have the `_total` suffix. Also the current number of slabs can be calculated with `count by(number)(memcached_slab_xxx)`, I would not export this as extra metric.